lnrpc/wtclientrpc/wtclient.go

```diff
@@ -462,11 +499,22 @@ func marshallTower(tower *wtclient.RegisteredTower, includeSessions bool,
 		}
 	}
 
-	return &Tower{
-		Pubkey:                 tower.IdentityKey.SerializeCompressed(),
-		Addresses:              rpcAddrs,
+	rpcTower := &Tower{
+		Pubkey:    tower.IdentityKey.SerializeCompressed(),
+		Addresses: rpcAddrs,
+		SessionInfo: []*TowerSessionInfo{{
+			PolicyType:             policyType,
+			ActiveSessionCandidate: tower.ActiveSessionCandidate,
+			NumSessions:            uint32(len(tower.Sessions)),
+			Sessions:               rpcSessions,
```

Comment:
nit: if we want to have deterministic order then `rpcSessions` should also be sorted by some criteria because it's populated from a `map` above where iteration is non-determinsitic.

Reply:
ahhhh yes good catch! silly me. Fixed!

Reply:
awsome!